week-2-practise/level-3/4-slice.js

```diff
@@ -5,7 +5,10 @@
 // - insert the replacement value at the provided index
 
 function replace(arr, index, value) {
-  return; // complete this statement
+  return arr
+    .slice(0, index)
+    .concat(value)
+    .concat(arr.slice(index + 1)); // complete this statement
 }
 
```

Comment:
Did you look at using `splice()` at all?

Reply:
i know that there is more than way to  solve the problem but because the problem title was slice so i used  slice  , I will try to use splice and post it here for your review .